.travis/after_success.sh

```diff
@@ -11,14 +11,14 @@ if [[ $UPLOAD == "no-upload" ]]; then
 else
     echo "Job with Conda upload..."
 
-    if [ x$TRAVIS_BRANCH = x"master" -a x$TRAVIS_EVENT_TYPE != x"cron" -a x$TRAVIS_PULL_REQUEST == xfalse ]; then
-    	$SPACER
-
-    	start_section "package.upload" "${GREEN}Package uploading...${NC}"
-    	anaconda -t $ANACONDA_TOKEN upload --user $ANACONDA_USER --label main $CONDA_OUT
-    	end_section "package.upload"
+    $SPACER
+    # Travis will not expose the ANACONDA_TOKEN var for pull requests coming from other forks than the original one
+    if [ x$ANACONDA_TOKEN != x ]; then
+        start_section "package.upload" "${GREEN}Package uploading...${NC}"
+        anaconda -t $ANACONDA_TOKEN upload --no-progress --user $ANACONDA_USER --label travis-$TRAVIS_BUILD_ID $CONDA_OUT
+        end_section "package.upload"
     else
-        echo "Conditions unmet, will not upload"
+        echo "Cannot upload packages from branches not uploaded to the original repository"
```

Comment:
I think we want to say something like "Please define ANACONDA_TOKEN in your travis build settings (url)"? or something similar?

Reply:
This is not really true. We have this defined, but it's a secret variable. Travis will not expose it to any branch from another fork. It's a security feature. There's nothing people could do. It's a serious problem from the development perspective.

Reply:
On a PR the token won't be available, but when people push to their own fork the tool should be uploading to that user's own anaconda repo which they can get an `ANACONDA_TOKEN` for.

Setting up their own anaconda repo and `ANACODNA_TOKEN` should be explained somewhere....

Reply:
understood